flang/test/lit.cfg.py

```diff
@@ -87,6 +87,9 @@
 # the build directory holding that tool.
 tools = [
   ToolSubst('%f18', command=FindTool('f18'),
+    extra_args=["-intrinsic-module-directory "+config.flang_intrinsic_modules_dir],
```

Comment:
That sounds good, the only problem is that the test I have for this patch would fail without the change in lit.cfg.py. I'm not sure what the best approach would be because of that. Should I have this change in a separate patch, remove the test for this patch and add it later when lit.cfg.py is upstreamed? Is there a better approach?

Reply:
You can add this entry for bbc in lit.cfg.py as a separate patch in fir-dev and if necessary (the bbc tool is not available upstream) upstream also.

Reply:
Alright, I'll make a separate patch in fir-dev for this entry